presto-docs/src/main/sphinx/presto_cpp/features.rst

```diff
@@ -314,6 +314,16 @@ If set to ``true``, disables the optimization in expression evaluation to delay
 
 This should only be used for debugging purposes.
 
+``native_selective_nimble_reader_enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Temporary flag to control whether selective Nimble reader should be used in this
+query or not.  Will be removed after the selective Nimble reader is fully rolled
```

Comment:
```suggestion
query or not.  
```
In general, technical documentation should avoid future tense. Describe the current state of the code. When the selective Nimble reader is fully rolled out and this temporary flag is removed, we can delete the doc along with the flag.